### PR TITLE
Release 2.10.13

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Review by krauselukas — APPROVED** [review]

**Review by danidoni — APPROVED** [review]

File: ReleaseNotes-2.10.13 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* Frontend:
+  - Fix XML external entity (XXE) injection in xmlhash
+    CVE-2022-21949
+  - Update to Ruby 2.7
+  - Fix heap memory corruption in yajl-ruby gem
+    https://github.com/brianmario/yajl-ruby/security/advisories/GHSA-jj47-x69x-mxrm
+  - Fix excessive backtracking in nokogiri gem
+    https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-crjr-9rc5-ghw8
+  - Fix priviledge escalation issue in ProjectDoProjectReleaseJob (#12407)
```
Comment (saraycp) [inline ReleaseNotes-2.10.13:44]:
```suggestion
  - Fix privilege escalation issue in ProjectDoProjectReleaseJob (#12407)
```

**Review by saraycp — CHANGES_REQUESTED** [review]

File: ReleaseNotes-2.10.13 [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+The dist/README.UPDATERS file has information for people updating
+from a previous OBS release.
+
+Updating from OBS 2.10.12
+=========================
+
+We have updated the ruby interpreter which requires a manual step when updating
+from a previous OBS version:
+
+  1) Change to passenger ruby to ruby2.7
```
[the remark this answers is not in the document]
Comment (hennevogel) [reply ReleaseNotes-2.10.13]: fixed

File: ReleaseNotes-2.10.13 (hunk 1 above)
Comment (hennevogel) [reply ReleaseNotes-2.10.13]: fixed

**Comment by hennevogel** [comment]
@saraycp can you have a look again please?

**Pull request merged**